src/Core/Session/Factory/Session.php

```diff
@@ -82,10 +80,15 @@ public function createSession(App\Mode $mode, App\BaseURL $baseURL, IManageConfi
 							$handler = new Handler\Cache($cache, $logger);
 						}
 						break;
+					default:
+						$handler = null;
```

Comment:
This should log an exception and set the sessionhandler to default

```suggestion
						$handler = self::HANDLER_DEFAULT;
						$this->logger->alert(...);
```

Reply:
`$handler` is supposed to be an object while `self::HANDLER_DEFAULT` is a string, I'm not sure how you reconcile this.